src/sidebar/components/group-list-item.js

```diff
@@ -85,9 +90,11 @@ function GroupListItem({
       onToggleSubmenu={toggleSubmenu}
       submenu={
         <Fragment>
-          <ul>
+          {/* Giving the role="menu" allows screen readers to have
+          correct number of subitems */}
+          <ul role="menu">
```

Comment:
I noticed chromevox does not read the correct # of subitems unless we have a second `role="menu"` here.

Reply:
I think it would be preferable to put this on the `.menu-item__submenu` div if possible, otherwise each time a dev authors a new component that renders submenus they will need to know to do this.